extensions/amp-auto-ads/0.1/opt-in-status.js

```diff
@@ -23,5 +23,4 @@ export const OptInStatus = {
   OPT_IN_STATUS_BANNER_ADS: 1,
```

Comment:
Is this file used any more? Can it just be deleted.

The MD file should document the API: https://github.com/ampproject/amphtml/blob/master/extensions/amp-auto-ads/amp-auto-ads.md. (although it seems the opt-in-status field is missing).

Reply:
Only used at one place. Doesn't necessary this enum field for that. Making it as it was before.